lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -393,24 +393,25 @@ sub job_next_previous_ajax {
 
 sub _calculate_preferred_machines {
```

Comment:
Can we add a unit test for this somewhere? This looks to be completed untested? At least in the unit tests I can't find it...

Reply:
It seems to be covered by tests somehow at least implicitly because codecov reports for the diff to be completely covered :)